common4j/src/main/com/microsoft/identity/common/java/nativeauth/authorities/NativeAuthCIAMAuthority.kt

```diff
@@ -104,7 +104,6 @@ class NativeAuthCIAMAuthority (
             correlationId = null,
             "${TAG}.getChallengeTypesWithDefault"
         )
-        Logger.info(TAG, "Challenge Types passed = $challengeTypes")
```

Comment:
Why was this removed? I'd say this is helpful debugging information.

Reply:
```kotlin
val challengeTypesWithDefault = (challengeTypes ?: emptyList()).plus(listOf(NativeAuthConstants.GrantType.REDIRECT)).distinct().joinToString(" ")
Logger.info(TAG, "Challenge Types used = $challengeTypesWithDefault")
return challengeTypesWithDefault
```
Put it back and use challengeTypesWithDefault